amivapi/auth/templates/loginpage.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <body>
+    <h1>{{client_id}} requests access to your AMIV account.</h1>
+    {% if error_msg %}
```

Comment:
Ideally, errors are displayed at the corresponding input field ;)
Also, the error messages defined earlier corresponding to wrong client app IDs are not very helpful to the user. It should add in these cases a remark to contact the developer of the client app.